react/training-project/src/redux/Auth/actions.js

```diff
@@ -36,6 +37,19 @@ const privateActionsCreators = {
   logOutSuccess: () => ({
     type: actions.LOG_OUT_SUCCESS,
     payload: null
+  }),
+
+  setUserData: ({ name, photoUrl, charge, description, won, lost, draw }) => ({
```

Comment:
Can't you generate the object before calling setUserData? So you can just do something like this:
```
setUserData: payload => ({
  type: actions.SET_USER_DATA,
  payload
})
```